.travis.yml

```diff
@@ -18,6 +18,11 @@ cache:
 before_script:
   - NODE_ENV=$BUILD_ENV npm run build
```

Comment:
I had removed this at first, thinking that all client stuff would be built as part of the `mocha-webpack` command,  but that gave me `Error: Cannot find module '../public/manifest.json'` 💩 

That means the server still relies on client-side build. Also that time saved on Travis CI won't be as nice. But surely that's something we can fix later :)